### PR TITLE
ISSUE-524 Create a source distribution module

### PR DESCRIPTION
**Pull request by cunningt**
https://github.com/apache/camel-kafka-connector/issues/524

### CODE REVIEW

File: distribution/src/main/assemblies/source-release.xml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<assembly>
+  <id>source-release</id>
+  <!-- the only thing being done differently here is the base directory -->
+  <baseDirectory>apache-camel-quarkus-${version}</baseDirectory>
```
Comment (oscerd) [inline distribution/src/main/assemblies/source-release.xml:23]: I think this needs to be related to kafkaconnector

**Review by oscerd — CHANGES_REQUESTED** [review]
Just one finding.

**Review by valdar — APPROVED** [review]

File: distribution/src/main/assemblies/source-release.xml (hunk 1 above)
Comment (cunningt) [reply distribution/src/main/assemblies/source-release.xml]: Fixed!

**Review by oscerd — APPROVED** [review]

**Comment by oscerd** [comment]
Thanks

**Pull request merged**